src/patternfly/upgrade-examples/AlertUpgradeExamples/examples/alert-upgrade-example-pf4-notification.hbs

```diff
@@ -0,0 +1,47 @@
+{{#> alert-upgrade-examples}}
+{{#> alert alert--modifier="pf-m-success" alert--attribute='aria-label="Success Notification"'}}
+  {{#> alert-icon alert-icon--success="true"}}
+  {{/alert-icon}}
+  {{#> alert-body}}
+    {{#> alert-description}}
+			The server configuration changed.
+		{{/alert-description}}
+	{{/alert-body}}
+{{/alert}}
+<br>
+{{#> alert alert--modifier="pf-m-danger" alert--attribute='aria-label="Danger Notification"'}}
+  {{#> alert-icon alert-icon--danger="true"}}
+  {{/alert-icon}}
+  {{#> alert-body}}
+    {{#> alert-description}}
+			Failed to add server_abc.
+		{{/alert-description}}
+  {{/alert-body}}
+{{/alert}}
+<br>
+{{#> alert alert--modifier="pf-m-warning" alert--attribute='aria-label="Warning Notification"'}}
+  {{#> alert-icon alert-icon--warning="true"}}
+  {{/alert-icon}}
+  {{#> alert-body}}
+    {{#> alert-description}}
+			The server configuration changed. 
+		{{/alert-description}}
+		{{#> alert-link alert-link--modifier="pf-c-button pf-m-link"}}
+			Reload server.
+		{{/alert-link}}
```

Comment:
I know I'm suggesting that this shouldn't go here. But this is more of a general question about how we are supposed to handle links. In a case like this, where a link is included with text, would we want to just use `<a>` without the button component classes? I noticed that when the link wraps to the next line, there's some awkward spacing.

Reply:
I think it would depend on whether the "reload server" performs an action (button) or takes you to a different page (link). If this were a link, I would change it to an `<a>`, but if it's a button, I would be inclined to get rid of this element and make "reload server" the button on the right edge of the alert.

Reply:
I missed this before, but when the Alert component displays dynamically, it should have `role="alert"` as described in the a11y section:

Attribute | Applied To | Outcome
-- | -- | --
role="alert" | .pf-c-alert__body | Communicates contents of the alert message to the user. This should only be used on dynamically created alerts, such as toast notifications or inline notifications that aren't present during page load (for example, an alert at the top of a form when there are errors in the form).

Also, @mcarrano and I were chatting last week about the best way to incorporate actions into these components, and will have to circle back to this one in a separate issue most likely, but I do know that we don't want to include clickable elements inside the element with `role="alert"`. For now, I'd say @michael-coker's suggestion is a good way to go.

Reply:
added role="alert" to pf-c-alert__body